src/components/TopBar/OldUIAnnouncement.tsx

```diff
@@ -0,0 +1,41 @@
+import { Alert, AlertActionCloseButton, AlertVariant } from '@patternfly/react-core';
+import { ApplicationsIcon, ArrowIcon } from '@patternfly/react-icons';
+import { useState } from 'react';
+
+const PNC_OLD_UI_URL_TRIGGER = 'secondary';
+
+export const OldUIAnnouncement = () => {
+  const [isClose, setIsClose] = useState<boolean>(false);
+  const oldUIUrl = process.env.REACT_APP_PNC_OLD_UI_WEB;
+
+  /* Only display the announcement when old ui is set to secondary */
+  const shouldDisplay = !!(oldUIUrl && oldUIUrl.includes(PNC_OLD_UI_URL_TRIGGER));
```

Comment:
Isn't better to use a condition that checks if the current browser URL does not contain 'secondary'? Then we could merge everything in advance and it would be automatically displayed once backend changes are done.